server/v2/cometbft/server.go

```diff
@@ -61,20 +65,39 @@ type App[T transaction.Tx] interface {
 	GetStore() types.Store
 }
 
-func NewCometBFTServer[T transaction.Tx](
-	app *appmanager.AppManager[T],
-	store types.Store,
-	logger log.Logger,
-	cfg Config,
-	txCodec transaction.Codec[T],
-) *CometBFTServer[T] {
+func New[T transaction.Tx](home string, txCodec transaction.Codec[T]) *CometBFTServer[T] {
+	// Write default cmt config
```

Comment:
I think we can only do that at Init/Start, as we don't have the home flag value here.